packages.dhall

```diff
@@ -57,6 +57,70 @@ let additions =
           ]
           "https://github.com/awakesecurity/purescript-precise-datetime.git"
           "v5.1.1"
+      , higher-order =
```

Comment:
You may be able to add these to https://github.com/purescript/package-sets so that they don't need to be maintained as explicit overrides in the future. Not a problem for this PR, but that makes things a little more convenient for folks downstream, as they can then just do `spago install <lib>`.